NitroxClient/MonoBehaviours/Gui/Chat/PlayerChatInputField.cs

```diff
@@ -153,6 +154,12 @@ private void Update()
                     FreezeTime = true;
                 }
             }
+
+            // Reselect chat if Y is pressed
+            if (UnityEngine.Input.GetKeyDown(KeyCode.Y))
```

Comment:
The chat key can be changed in the options menu, so you need to get that value instead of fixed "Y"

Reply:
Thinking about it when I'm not tired, The actual action for this exists within the ChatKeyBindingAction file,

The limitation on opening may come from the blocking if statement, so I'll experiment by removing that blocking if